builtin/game/falling.lua

```diff
@@ -42,8 +42,7 @@ core.register_entity(":__builtin:falling_node", {
 
 	on_step = function(self, dtime)
 		 -- Set gravity
-		local acceleration = self.object:getacceleration()
-		if not vector.equals(acceleration, {x = 0, y = -10, z = 0}) then
+		if self.object:getacceleration().y ~= -10 then
```

Comment:
I'm not sure why this is even done in `on_step()` at all. It seems the acceleration needs to be done only once in the activation, and this entire check can then be removed here.